apollo-router/src/query_planner/bridge_query_planner_pool.rs

```diff
@@ -116,12 +116,11 @@ impl BridgeQueryPlannerPool {
                             continue;
                         }
                     };
-                    let start = Instant::now();
 
                     let res = svc.call(request).await;
 
                     f64_histogram!(
-                        "apollo.router.query_planning.plan.duration",
+                        "apollo.router.query_planning.pooled_planner.duration",
```

Comment:
We try as much as we can to not rename an existing metric because it could break existing dashboard of our customers and would introduce a breaking change

Reply:
Hmm I just read the PR's description more in details. And it looks like this is like a different one and the previous one does still exist... Which is valid.

cc @BrynCooke do you have any opinion ?

Reply:
Yeah, we can't rename the existing metric.